<launch>
    <arg name="robot_name" default="small_scout_1" />
    <arg name="output" default="screen" />
    <arg name="use_cheat_odom" default="true" />
    <arg name="use_crab_drive" default="false" />

    <include file="$(find operations)/launch/navigation.launch">
        <arg name="robot_name" value="$(arg robot_name)" />
        <arg name="output" value="$(arg output)" />
        <arg name="use_cheat_odom" value="$(arg use_cheat_odom)" />
        <arg name="use_crab_drive" value="$(arg use_crab_drive)" />
    </include>

    <group ns="/capricorn/$(arg robot_name)">        
        <node pkg="state_machines" type="start_scout_sm" name="scout_1_sm" args="$(arg robot_name)" output="screen" />
        <node name="navigation_vision_server" pkg="operations" type="navigation_vision_server" args="$(arg robot_name)" output="screen"/>
        <node pkg="operations" type="resource_localiser" name="scout_1_localiser" args="$(arg robot_name)" output="screen" />
        <node pkg="operations" type="scout_search" name="scout_1_search" args="$(arg robot_name)" output="screen" />
    </group>
</launch>